packages/core/src/new-api/internal/types/execution-state.ts

```diff
@@ -125,8 +125,7 @@ export interface DeploymentExecutionState
     | FutureType.NAMED_LIBRARY_DEPLOYMENT
     | FutureType.ARTIFACT_LIBRARY_DEPLOYMENT
   > {
-  storedArtifactPath: string; // As stored in the deployment directory.
-  storedBuildInfoPath?: string; // As stored in the deployment directory. Optional as it's not always present
+  artifactFutureId: string; // As stored in the deployment directory.
```

Comment:
We may want to revisit this later, minimizing the state that we store, and relying more on "view" functions. This would keep the model and reducers simpler. Fine for now though.

Reply:
Recorded as a cleanup task here: https://github.com/NomicFoundation/ignition/issues/322